Documentation/gh-pages/source/introduction.rst

```diff
@@ -17,7 +17,7 @@ The Kasper framework
 ====================
 
 The *Kassiopeia* software relies upon three separate, but concurrently distributed libraries. These are *Kommon*,
-*KGeoBag*, and *KEMField* and part of the **Kasper** framework.
+*KGeoBag*, and *KEMField* and are part of the **Kasper** framework.
```

Comment:
This sentence was perfectly fine, since it is just an enumeration :)

```suggestion
*KGeoBag*, and *KEMField* and part of the **Kasper** framework.
```

Reply:
Well, they are no longer part of the **Kasper** framework, so how about this?

```suggestion
*KGeoBag* and *KEMField*.
```

Reply:
The Table of Contents is still visible in the bar on the left?

Reply:
That's right and I like the suggestion.